packages/@aws-cdk-containers/ecs-service-extensions/lib/service.ts

```diff
@@ -10,6 +10,17 @@ import { ServiceDescription } from './service-description';
 // eslint-disable-next-line no-duplicate-imports, import/order
 import { Construct } from '@aws-cdk/core';
 
+/**
+ * connectToProps will have all the extra parameters which are required for connecting services.
+ */
+export interface connectToProps {
+  /**
+   * local_bind_port is the local port that this application should
+   * use when calling the upstream service in ECS Consul Mesh Extension
+   */
+  readonly local_bind_port?: number;
```

Comment:
We should probably update this comment to be more generic and not specific to the ECS Consul Mesh Extension.

Reply:
But this particular parameter will be only used in the consul service mesh and not in any other mesh for example AppMesh.